File: 260112/정수 사이의 배수/multiplication-between-numbers.js
const fs = require('fs');
const [a, b] = fs.readFileSync(0).toString().trim().split(' ').map(Number);

let sum = 0;
let num = 0;

for (let i = a; i <= b; i++) {
    if (i % 5 === 0 || i % 7 === 0) {
        sum += i;
        num++;
    }
}

console.log(sum, (sum / num).toFixed(1));

File: 260112/정수 사이의 배수/multiplication-between-numbers.py
a, b = map(int, input().split())
total = 0
num = 0

for i in range(a, b + 1):
    if i % 5 == 0 or i % 7 == 0:
        total += i
        num += 1

print(total, round(total / num, 1))